Instagram reels and post regex fixed

Abhii5496 opened the pull request:
with slight modification in regex of instagram , now it will render post and reel without any issue

neSpecc reviewed (changes requested):
Please, remove unrelated changes.
Describe issues you're resolving in a PR description.
Give a few URL to check

Abhii5496 commented:
> Please, remove unrelated changes. Describe issues you're resolving in a PR description. Give a few URL to check

sure

Abhii5496 commented:
i have removed the unrelated changes and added description
here are some urls to check ::
https://www.instagram.com/p/C4_Lsf1NBra/?img_index=1
https://www.instagram.com/p/C5ZZUWPydSY/?utm_source=ig_web_copy_link
https://www.instagram.com/reel/C19IuqJx6wm/
https://www.instagram.com/reel/C19IuqJx6wm/?utm_source=ig_web_copy_link

neSpecc reviewed (changes requested):
Please add a corresponded tests

Abhii5496 commented:
> Please add corresponded tests and update a patch version

added the test , can you please check ,

neSpecc reviewed (commented):
update a patch version please

Abhii5496 commented:
@neSpecc , updated it .

neSpecc commented:
Resolve conflicts, please

Abhii5496 commented:
@neSpecc conflict resolved , merge it.

neSpecc reviewed (commented):
increment a patch version please
